libs/novui/src/json-schema-components/templates/IconButton.tsx

```diff
@@ -1,26 +1,52 @@
 import { IconButtonProps } from '@rjsf/utils';
-import { css } from '../../../styled-system/css';
+import { ButtonHTMLAttributes, FC } from 'react';
+import { IconType } from '../../icons';
+import { css, cx } from '../../../styled-system/css';
 import { Button, IconButton } from '../../components';
 import { IconAdd, IconArrowDownward, IconArrowUpward, IconOutlineDeleteOutline } from '../../icons/icon-registry';
 
 type Props = Omit<IconButtonProps, 'color' | 'translate' | 'iconType' | 'icon'>;
 
+const SimpleIconButton: FC<ButtonHTMLAttributes<HTMLButtonElement> & { Icon: IconType }> = ({
+  Icon,
+  title,
+  ...buttonProps
+}) => (
+  <button className={css({ cursor: 'pointer', _hover: { opacity: 'hover' } })} {...buttonProps}>
+    <Icon title={title} />
+  </button>
+);
+
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** ‏Delete this